src/test/regress/expected/multi_limit_clause.out

```diff
@@ -521,6 +521,63 @@ SELECT
           1 |    1
 (1 row)
 
+-- check if we can correctly push the limit when it is null
+SELECT l_orderkey FROM lineitem WHERE l_orderkey < 3 ORDER BY l_orderkey LIMIT null;
+DEBUG:  push down of limit count: ALL
+ l_orderkey
+---------------------------------------------------------------------
+          1
+          1
+          1
+          1
+          1
+          1
+          2
+(7 rows)
+
+SELECT l_orderkey FROM lineitem WHERE l_orderkey < 3 ORDER BY l_orderkey OFFSET 1 LIMIT null;
```

Comment:
Do you think that it makes sense to add a test case with `LIMIT ALL` to show that we cover this as well?

Reply:
Added two tests, it looks like we did not have null/all tests in this file.